app/src/main/java/pl/hypeapp/wykopolka/network/api/WykopolkaApi.java

```diff
@@ -115,6 +116,11 @@
                                           @Field("receiverid") int receiverId, @Header("apisign") String apisign);
 
     @FormUrlEncoded
+    @POST("api/request/user/notify/call")
+    Observable<ResponseBody> callUserOnMikroblog(@Field("accountkey") String accountKey, @Field("wishid") String wishId,
```

Comment:
`Signle` should be better. Read differences between `Signle` and `Observerable`, but the main difference is that Single emit only one item instead `Observable` who is emits many items.